v0.5 requires rustc v1.21.0 minimum

I see the readme says 1.17 is required, but it looks like `unicode-normilization` has been updated to now require 1.21.0 minimum `rustc`.

To reproduce:

```
$ git clone https://github.com/mgeisler/version-sync
[.. snip ..]
$ cd version-sync
$ rustup override add 1.20.0
$ cargo build
   Compiling unicode-normalization v0.1.7
error[E0597]: borrowed value does not live long enough
    --> /home/kevin/.cargo/registry/src/github.com-1ecc6299db9ec823/unicode-normalization-0.1.7/src/tables.rs:7533:30
     |
7533 |         '\u{1F248}' => Some(&['\u{3014}', '\u{6557}', '\u{3015}']),
     |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^- temporary value only lives until here
     |                              |
     |                              does not live long enough
     |
     = note: borrowed value must be valid for the static lifetime...

error[E0597]: borrowed value does not live long enough
    --> /home/kevin/.cargo/registry/src/github.com-1ecc6299db9ec823/unicode-normalization-0.1.7/src/tables.rs:7534:30
     |
7534 |         '\u{1F250}' => Some(&['\u{5F97}']),
     |                              ^^^^^^^^^^^^- temporary value only lives until here
     |                              |
     |                              does not live long enough
     |
     = note: borrowed value must be valid for the static lifetime...

error[E0597]: borrowed value does not live long enough
    --> /home/kevin/.cargo/registry/src/github.com-1ecc6299db9ec823/unicode-normalization-0.1.7/src/tables.rs:7535:30
     |
7535 |         '\u{1F251}' => Some(&['\u{53EF}']),
     |                              ^^^^^^^^^^^^- temporary value only lives until here
     |                              |
     |                              does not live long enough
     |
     = note: borrowed value must be valid for the static lifetime...

error: aborting due to 5738 previous errors

error: Could not compile `unicode-normalization`.

To learn more, run the command again with --verbose.
$ rustup override add 1.21.0
$ cargo build
[.. snip ..]
   Compiling unicode-normalization v0.1.7
    Finished dev [unoptimized + debuginfo] target(s) in 25.18 secs
```

--- a/README.md
+++ b/README.md
@@ -116,9 +116,6 @@ This is a changelog describing the most important changes per release.
 
 ### Unreleased
 
-Dependencies were updated and `version-sync` now requires Rust version
-1.21 or later.
-
 Git dependencies are now always accepted, which means that blocks like
 
 ~~~markdown
@@ -133,12 +130,13 @@ will work without you having to add `no_sync`.
 Issues closed:
 
 * [#42][issue-42]: Handle Git dependencies
+* [#39][issue-39]: Version 0.5 requires Rust version 1.21.0
 
 
 ### Version 0.5.0 — November 19th, 2017
 
 Dependencies were updated and `version-sync` now requires Rust version
-1.17 or later.
+1.21 or later.
 
 Error messages from `assert_html_root_url_updated` now again include
 line numbers (based on a heuristic until the `syn` crate can provide
@@ -201,4 +199,5 @@ Contributions will be accepted under the same license.
 [codecov]: https://codecov.io/gh/mgeisler/version-sync
 [mit]: LICENSE
 [issue-19]: https://github.com/mgeisler/version-sync/issues/19
+[issue-39]: https://github.com/mgeisler/version-sync/issues/39
 [issue-42]: https://github.com/mgeisler/version-sync/issues/42

